test/resolver.spec.js

```diff
@@ -146,6 +146,15 @@ describe('IPLD Format resolver (local)', () => {
         })
       })
 
+      it('links by name', (done) => {
+        resolver.resolve(linksNodeBlob, 'named link', (err, result) => {
+          expect(err).to.not.exist()
+          expect(result.value['/']).to.eql(links[1].multihash)
+          expect(result.remainderPath).to.eql('')
+          done()
+        })
+      })
+
```

Comment:
Could you please add tests for:
 - named link and a remainder path
 - error case for non-existent named link